IBMigrationTool/InterfaceBuilderFileMigrator.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  InterfaceBuilderFileMigrator.swift
+//  ThunderBasics
+//
+//  Created by Simon Mitchell on 18/09/2020.
+//  Copyright © 2020 threesidedcube. All rights reserved.
+//
+
+import Foundation
+
+/// A class responsible for migrating an Interface Builder to fix issues
+/// when upgrading to `ThunderBasics` v2.0.0
+final class InterfaceBuilderFileMigrator {
+    
+    static let directlyMappableAttributes: [(String, String)] = [
+        ("color", "borderColor"),
+        ("number", "borderWidth"),
+        ("number", "cornerRadius"),
+        ("color", "shadowColor"),
+        ("number", "shadowOpacity"),
+        ("number", "shadowRadius")
+    ]
+    
+    static let removedCustomClasses: [String] = [
+        "TSCTextView",
+        "TSCView",
+        "TSCImageView"
+    ]
+    
+    /// Represents the current contents of the interface builder file
+    ///
+    /// - Note: This will be changed by calling the `migrate` function to
+    /// represent the fixed version of the file!
+    var string: String
+    
+    convenience init?(filePath: String) {
+        let fileURL = URL(fileURLWithPath: filePath)
+        guard let data = try? Data(contentsOf: fileURL) else {
+            return nil
+        }
+        guard let fileString = String(data: data, encoding: .utf8) else {
+            return nil
+        }
+        self.init(string: fileString)
+    }
+    
+    init(string: String) {
+        self.string = string
+    }
+    
+    /// Performs migration of the current file, storing the result in `string`
+    func migrate() {
+        migrateUserDefinedRuntimeAttributes()
+        migrateRemovedCustomClasses()
+    }
+    
+    private func migrateRemovedCustomClasses() {
+        
+        Self.removedCustomClasses.forEach { (customClass) in
+            
+            string = string.replacingOccurrences(
+                of: "\\s+customClass=\"\(customClass)\"\\s?(customModule=\"ThunderBasics\")?",
```

Comment:
Does the `customModule` always come after the `customClass`? 😄.

Reply:
Good question... not entirely sure! Reckon I should add it in the other way too? :)

Reply:
@simonmitchell I guess there's not really anything to lose by adding it in, performance is not an issue here

Reply:
Done :D